Apps/WebClient/src/ClientApp/src/views/ServicesView.vue

```diff
@@ -0,0 +1,46 @@
+﻿<script lang="ts">
+import Vue from "vue";
+import { Component } from "vue-property-decorator";
+
+import LoadingComponent from "@/components/LoadingComponent.vue";
+import BreadcrumbComponent from "@/components/navmenu/BreadcrumbComponent.vue";
+import BreadcrumbItem from "@/models/breadcrumbItem";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const options: any = {
+    components: {
+        BreadcrumbComponent,
+        LoadingComponent,
+    },
+};
+@Component(options)
+export default class ServicesView extends Vue {
+    private isLoading = true;
```

Comment:
Recommend making isLoading into a getter since that will allow easy updating if it needs to check multiple properties in the future:

```
get isLoading(): boolean {
    return false;
}
```